### PR TITLE
[Ongoing]  Feat/1205292031849977 rename peaq dev

### PR DESCRIPTION
**Pull request by sfffaaa**
1. Rename peaq-dev to agung
2. Increase the version of the runtime module
3. Extract some utils in the node module

Note:
Keep the currencyId (PEAQ/DOT) the same as before, and will change that in the related task

### CODE REVIEW

File: pallets/block-reward/src/weights.rs [hunk 1]
```diff
@@ -3,14 +3,14 @@
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
 //! DATE: 2022-09-29, STEPS: `20`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
-//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("agung"), DB CACHE: 1024
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
```
Comment (neutrinoks) [inline pallets/block-reward/src/weights.rs:6]: So far, every "dev" was replaced by "agung", here it is the other way round :upside_down_face:

File: pallets/block-reward/src/weights.rs [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
 
 // Executed Command:
 // ./target/release/peaq-node
 // benchmark
 // pallet
 // --chain
-// agung
+// dev
```
Comment (neutrinoks) [inline pallets/block-reward/src/weights.rs:13]: Same here...

File: runtime/agung/src/lib.rs [hunk 3]
```diff
@@ -433,7 +432,8 @@ type NegativeImbalance = <Balances as Currency<AccountId>>::NegativeImbalance;
 
 parameter_types! {
 	pub PcpcLocalAccepted: Vec<CurrencyId> = vec![
-		CurrencyId::Token(TokenSymbol::ROC),
```
Comment (neutrinoks) [inline runtime/agung/src/lib.rs:436]: Don't we accept the ROC-token?! :thinking: the Agung parachain is usually connected to Rococo, not Polkadot... is this correct? I think, we should accept ROC as token from relaychain...

File: runtime/common/src/lib.rs [hunk 4]
```diff
@@ -203,8 +203,7 @@ where
 			MultiLocation { parents: 1, interior: Here } => {
 				let version = <T as SysConfig>::Version::get();
 				match version.spec_name {
-					RsBorrowed("peaq-node-dev") => Some(Token(DOT)),
-					RsBorrowed("peaq-node-agung") => Some(Token(ROC)),
+					RsBorrowed("peaq-node-agung") => Some(Token(DOT)),
```
Comment (neutrinoks) [inline runtime/common/src/lib.rs:206]: Here also the DOT<->ROC topic... I think it should be ROC

**Comment by sfffaaa** [comment]
Close this because we have several modifications. After merging the other features' implementation, will be back here to do the rename again

**Pull request closed without merging**